Gemfile

```diff
@@ -49,6 +49,10 @@ gem 'rails_12factor'
 
 gem "virtus"
 
+# For proper email rendering in Gmail
+gem 'nokogiri'
```

Comment:
Nokogiri is a [dependency of `premailer-rails`](https://github.com/fphilipe/premailer-rails#installation)